### PR TITLE
Fix cartopy

### PR DESCRIPTION
**Pull request by magnarem**
**Summary**:

**Related issue**:

**Suggested reviewer(s)**:

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

**Comment by magnarem** [comment]
This PR fixes multiple issues.

- Fixing issue #169 after removing cartopy as dependency from sol-indexer.
- Fixes #162. Removed debug logging that was used for testing code, and not needed.
- Fixing #4 from github.com/metno/solr-indexer

**Comment by magnarem** [comment]
Ready for merge CI containers now working

File: dmci/distributors/solr_dist.py [hunk 1]
```diff
@@ -26,6 +26,11 @@
 
 logger = logging.getLogger(__name__)
 
+"""Configure log level for solrindexer
+NOTE: Should the be configureable from dmci config?
```
Comment (mortenwh) [inline dmci/distributors/solr_dist.py:30]: The solrindexer will be installed in the dmci container, right? Then I guess the answer is yes.. We only have one log for the container, right?

File: dmci/distributors/solr_dist.py [hunk 2]
```diff
@@ -129,8 +134,11 @@ def _index_record(self, newdoc, add_thumbnail=False, level=1):
         try:
             status, msg = self.mysolr.index_record(
                 newdoc, addThumbnail=add_thumbnail, level=level)
+            logger.info("Indexed document: %s"
```
Comment (mortenwh) [inline dmci/distributors/solr_dist.py:137]:
```suggestion
            logger.info("Indexed document %s in SolR"
```

File: dmci/distributors/solr_dist.py [hunk 3]
```diff
@@ -141,7 +149,8 @@ def _delete(self):
                                                 self._metadata_id)
         status, msg = self.mysolr.delete(
             identifier, commit=self._conf.commit_on_delete)
-        logger.debug("Delete status: " + str(status) + ". With response: " + str(msg))
+        logger.info("Delete status: %s. With response: %s" %
```
Comment (mortenwh) [inline dmci/distributors/solr_dist.py:152]:
```suggestion
        logger.info("SolR delete status: %s. With response: %s" %
```

File: dmci/api/worker.py [hunk 4]
```diff
@@ -124,13 +123,16 @@ def validate(self, data):
...
+                    found_parent_block_content = found_parent_block_content.split(
+                        b":")
                     if len(found_parent_block_content) != 2:
                         err = f"Malformed parent dataset identifier {found_parent_block_content}"
                         logger.error(err)
                         return False, err, data
-                    old_parent_namespace = found_parent_block_content[0].decode()
-                    logger.debug("Parent dataset namespace: %s" % old_parent_namespace)
+                    old_parent_namespace = found_parent_block_content[0].decode(
+                    )
```
Comment (mortenwh) [inline dmci/api/worker.py:133]:
```suggestion
                    old_parent_namespace = found_parent_block_content[0].decode()
```

**Review by mortenwh — CHANGES_REQUESTED** [review]
Foreslår å skrive eksplisitt SolR i logg-meldingene, slik at det er tydelig når vi leser de i sammenhenge med pycsw- og filedist. Ser ellers bra ut!

**Pull request merged**